### ISSUE TITLE
Login Nodes Feature does not set `Tags:` 

### ISSUE THREAD

**Issue by michaelmayer2** [aws/aws-parallelcluster#5790]
When using `Tags:` and `LoginNodes` in the same cluster environment, the login node creation and as a consequence the whole cluster will fail s the launch template for the Login Nodes reports "Permission denied". 

This happens on an AWS account where the presence of tags is enforced. 

Tested with AWS ParallelCluster 3.7.2

Workaround: Disable automatic rollback and manually edit Launch template for Login Node creation in order to add the requested Tags there, then retry the creation of the full stack. This time it will succeed. 

Sample template file below. 

```
Region: eu-west-1
Image:
  Os: alinux2
HeadNode:
  InstanceType: t2.micro
  Networking:
    SubnetId: subnet-xyz
  Ssh:
    KeyName: michael
Scheduling:
  Scheduler: slurm
  SlurmQueues:
  - Name: queue1
    ComputeResources:
    - Name: t2micro
      Instances:
      - InstanceType: t2.micro
      MinCount: 0
      MaxCount: 10
    Networking:
      SubnetIds:
      - subnet-xyz

LoginNodes:
  Pools:
    - Name: login
      Count: 1 
      InstanceType: t2.xlarge
      Networking:
        SubnetIds: 
          - subnet-xyz

Tags:
  - Key: rs:environment
    Value: development
  - Key: rs:project
    Value: solutions
```

**Comment by davprat** [comment]
Thank you for reporting this! I've created task to track this internally.

**Comment by michaelmayer2** [comment]
@davprat I think this issue has been fixed in ParallelCluster 3.8.0 - at least I now can see the launch templates for the login nodes using the proper Tags. It would be great if you could verify from your end.

**Comment by enrico-usai** [comment]
HI @michaelmayer2 this has been fixed by this: https://github.com/aws/aws-parallelcluster/pull/5854 that is part of 3.8.0. We missed to add it in the changelog.
